backend/src/main/java/com/videolibrary/backend/infrastructure/rest/convert/SeriesMapper.java

```
@@ -0,0 +1,14 @@
+package com.videolibrary.backend.infrastructure.rest.convert;
+
+import com.videolibrary.backend.domain.entity.Series;
+import com.videolibrary.backend.infrastructure.rest.dto.SeriesDto;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = DefaultMapperConfig.class)
+public interface SeriesMapper {
+
+    @Mapping(target = "episode.season", ignore = true)
```

This looks quite interesting. Maybe episode dto does not need to have a season

---

I saw that it does not have. Is this ignore needed?

---

Actually this mapping is not needed. We can just delete season field from EpisodeDto and it will silently ignore it.